tests/smoke/smoke_test.go

```diff
@@ -58,5 +58,9 @@ func (suite *SmokeSuite) SetupSuite() {
 }
 
 func (suite *SmokeSuite) TearDownSuite() {
+	if suite.T().Failed() {
+		suite.k8sh.GetRookLogs("rook-operator", "default", "_output/tests/", suite.T().Name())
+		suite.k8sh.GetRookLogs("rook-api", "rook", "_output/tests/", suite.T().Name())
```

Comment:
Yes, I am still working on this, I forgot to put WIP on the PR.  I will include, osd,monitor,mds and rgw logs as well.